.eslintrc

```diff
@@ -44,6 +44,7 @@
     "react/react-in-jsx-scope": "off",
```

Comment:
I know this is not in this PR but lets keep the flags consistent and change this to a "0" rather than "off"

Reply:
Woohooo! 
I'll change